File: src/page/DefaultPage.java
package page;

public class DefaultPage extends Page {

    private static final long serialVersionUID = 1L;

    private final JBTN back = new JBTN(0, "back");

    protected DefaultPage(Page p) {
        super(p);

        ini();
        resized();
    }

    @Override
    protected void resized(int x, int y) {
        setBounds(0, 0, x, y);
        set(back, x, y, 0, 0, 200, 50);
    }

    private void addListeners() {
        back.setLnr(e -> changePanel(getFront()));
    }

    private void ini() {
        add(back);
        addListeners();
    }

}

File: src/page/DefaultPage.kt
package page

import page.Page
import java.awt.event.ActionEvent
import java.util.function.Consumer

class DefaultPage protected constructor(p: Page?) : Page(p) {
    private val back: JBTN = JBTN(0, "back")
    override fun resized(x: Int, y: Int) {
        setBounds(0, 0, x, y)
        Page.Companion.set(back, x, y, 0, 0, 200, 50)
    }

    private fun addListeners() {
        back.setLnr(Consumer { e: ActionEvent? -> changePanel(front) })
    }

    private fun ini() {
        add(back)
        addListeners()
    }

    companion object {
        private const val serialVersionUID = 1L
    }

    init {
        ini()
        resized()
    }
}
